parts/linux/cloud-init/artifacts/labels.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Update Labels for Kubernetes nodes
+After=kubelet.service
+[Service]
+Restart=always
+RestartSec=60
+ExecStart=/bin/bash /opt/azure/containers/labels.sh
```

Comment:
why do we need to preserve this? Does this mean we won't allow RP to add any label to the node? and customer cannot do so too?

Reply:
From my understanding the reasoning we want this is that it makes the nodes responsible for their own labels. RP could still add its own labels. However, I believe the labels being added in this way will become the standard option. @paulgmiller and @gtxistxgao should be able to confirm this information, or correct me where I am wrong

Reply:
> can you explain why do we need this to be a reconciler in node? do we not want cx to update node label at all?

Not sure if this thread gave you an explination for this or not. If not, hoping Paul/Frank can give a better explination

Reply:
We 
1) Don't want to stop the customer from adding their own labels 
2) Do want to update the api server with the customnodelabels at vm startup time (after a reimage in particular)(
3) Think it is good to constantly update/restore the customnodelabels but this is less crucial. So open to debate here.

Reply:
got you. Think I misunderstood the --overwrite function so the original question makes no sense. sorry for the confusion.